### ISSUE TITLE
Is there a way to change update information in .AppImage file without recreating it?

### ISSUE THREAD

**Issue by szymonk211** [AppImage/AppImageUpdate#163]

**Comment by probonopd** [comment]
No, that is not possible. AppImages are immutable by design. You need to unpack, make your changes, and repack the AppImage. This is a feature, not a bug.

**Comment by TheAssassin** [comment]
Of course it's possible, but none of our tools supports it, intentionally. It's a rare use case. With appimagetool, it's faster/easier to unpack, and repack with the correct information.

**Comment by szymonk211** [comment]
Thank you, it is not a big deal, I thought it would be possible because it was stated in documentation:
"this information is not put into the file system inside the AppImage, but rather embedded into the AppImage in a way that makes it very easy to change this information should it be required" - that's why I thought it is possible. Unpack and repack is a good way too.

**Comment by probonopd** [comment]
> this information is not put into the file system inside the AppImage

This applies to some metadata but not to the main compressed filesystem image.

**Comment by TheAssassin** [comment]
@szymonk211 can you provide a link, please? If it's in the _docs_, I'd like to clarify that. PRs also welcome.

**Comment by probonopd** [comment]
Sorry, I had answered too fast without reading the headline.
This is about __update information__.

Yes it can be changed without re-creating the AppImage.

**Comment by szymonk211** [comment]
Here is the link if you still need it:
https://github.com/AppImage/AppImageUpdate#update-information-overview

**Comment by szymonk211** [comment]
For me it was natural to search  for command line option that let someone change this metadata in script  https://github.com/AppImage/AppImageUpdate/blob/main/src/main.cpp. Maybe this is a good place to include this tool.
